control-plane/pkg/reconciler/channel/testdata/testdata

```diff
@@ -0,0 +1 @@
+testdata
```

Comment:
We broke some GH actions tests with this symlink :(

Reply:
Looking...